client/src/components/Button.js

```diff
@@ -0,0 +1,18 @@
+import styled from '@emotion/styled';
+
+const Button = styled.button`
+  background-color: #8645ff;
+  color: white;
+  text-align: center;
+  width: 190px;
```

Comment:
You shouldn't give your buttons a fixed width and height. It is better they take the size of their context and you style the padding. Think about the case, you want to use that button again, but with a long text in it. In this case, the text would be cut, because of the fixed button width.
I recommend having a look at Leon's comment in one of my [pull-requests](https://github.com/florianGierlichs/mycrypt/pull/24)

Reply:
Yes, that's true but I would like to have the same button size on every page for a uniform design independent of the amount of text on a button. Therefore the buttons are created wide so that it would also work with more text.
If the space would be too small at some point, this sentence would not belong in a button for me, because it would not fulfill the purpose of a button for me to have to insert a long explanatory sentence. 
Otherwise, would you know another solution with paddings, so that the buttons get the same size regardless of the content without a fixed size?

Reply:
A smart solution to make all buttons the same size is, by using a flexbox as a parent. You can then give all the children of that flexbox (in this case the buttons) the same size with flex-basis for example.
This is, how I solved the "same-size-button-problem":
```js
const ButtonContainer = styled.div`
  display: flex;
  justify-content: space-between;
  > * {
    flex-basis: 45%;
  }
`;
```

I hope this may help you 😄

Reply:
I absolutely have to think more in container and flexbox! I thank you for this great solution with flex-base! Now, it works even without fixed sizes :)